.github/workflows/test.yaml

```diff
@@ -58,3 +58,22 @@ jobs:
         run: |
           ct lint-and-install --target-branch ${{ github.event.repository.default_branch }} --upgrade --debug \
           --helm-extra-set-args '--set ci=true --set global.blobvault.persistentVolume.storageClassName="standard" --set imagePullSecrets[0].name=iterativeai --set dockerUsername=${{ secrets.ITERATIVE_DOCKER_REGISTRY_USER }} --set dockerPassword=${{ secrets.ITERATIVE_DOCKER_REGISTRY_PASSWORD }} --set dockerServer=docker.iterative.ai'
+
+  helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.2
+
+      - name: Run helm-docs
+        working-directory: charts/studio/
+        run: |
+          docker run --pul always --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest
```

Comment:
I checked if this does work [here](https://github.com/iterative/helm-charts/actions/runs/5555953172/jobs/10147815641)

![image](https://github.com/iterative/helm-charts/assets/5058132/76246090-94c2-4aa4-8103-794fe4338f0e)